api/api.go

```diff
@@ -343,6 +359,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 			continue
 		}
 
+		if !showInhibited && len(status.InhibitedBy) != 0 {
```

Comment:
still hoping that we'll one day have a real index around this :slightly_smiling_face: